apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/tag-term.e2e.cy.ts

```diff
@@ -0,0 +1,54 @@
+describe(`tagging a note`, () => {
+    const termBaseRoute = `/Resources/Terms/`;
+
+    const termId = `9b1deb4d-3b7d-4bad-9bdd-2b0d7b110003`;
+
+    const tagId = '9b1deb4d-3b7d-4bad-9bdd-2b0d7b110004';
+
+    const tagResourceLabel = `Tag Resource or Note`;
+
```

Comment:
I seem to be getting the same error here: 
<img width="552" alt="image" src="https://github.com/COSCRAD/coscrad/assets/63014783/574b598e-ef47-4551-9cb4-003b84ae2d28">

Reply:
as above